metaseq/checkpoint_utils.py

```diff
@@ -179,14 +179,41 @@ def _delete_old_checkpoint_files(
                 os.remove(old_chk)
 
 
+def verify_shards(cfg, dir=None, checkpoint_name=None):
```

Comment:
@KUNAL1612 This checkpoint verification code should already be checked in by now right? Why is this a part of this PR?

Reply:
I think because no one approved the previous PR so I couldn't merge it. I re-sent requests but am still waiting for approval

Reply:
Which previous PR is needed?

Reply:
#158 but I just merged it